src/core/Akka/Actor/ActorCell.cs

```diff
@@ -507,13 +541,12 @@ public static IActorRef GetCurrentSelfOrNoSender()
         }
 
         /// <summary>
-        /// TBD
+        /// Gets the current sender or <see cref="ActorRefs.NoSender"/>
         /// </summary>
-        /// <returns>TBD</returns>
         public static IActorRef GetCurrentSenderOrNoSender()
         {
             var current = Current;
-            return current != null ? current.Sender : ActorRefs.NoSender;
+            return current?.Sender ?? ActorRefs.NoSender;
```

Comment:
Since `Current` can be `null` and `Sender` can _separately_ be `null` (this has always been a possibility), we return `ActorRefs.NoSender` here - which, ironically, is `null` right now.